src/QMCHamiltonians/SOECPComponent.h

```diff
@@ -0,0 +1,118 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
```

Comment:
The copyright date should be updated, and the developed by/created by info should be update with a new file.

Reply:
Updated.